### ISSUE TITLE
Make scatter plots gallery example colourblind friendly

### ISSUE THREAD

**Issue by weiji14** [GenericMappingTools/pygmt#1011]
**Description of the problem**

The orange and green colours at the ["Scatter plots with a legend"](https://www.pygmt.org/v0.3.0/gallery/plot/scatter.html#sphx-glr-gallery-plot-scatter-py) gallery example can be hard to distinguish for people with red/green colour vision deficiencies. We should choose different colours instead if possible.

| [v0.3.0](https://www.pygmt.org/v0.3.0/gallery/plot/scatter.html#sphx-glr-gallery-plot-scatter-py) original colour | Deuteranopia (no green) simulation |
|--|--|
| ![original colour](https://user-images.githubusercontent.com/23487320/110191526-b52c4d00-7e8d-11eb-856c-efc4640799ad.png) | ![Deuteranopia simulation](https://user-images.githubusercontent.com/23487320/110191698-c3c73400-7e8e-11eb-85b8-61d1b2e99993.png) |

The line that needs to change is this:

https://github.com/GenericMappingTools/pygmt/blob/65b099a5873b4f9e76d7fccdbc8882d9b37870e3/examples/gallery/plot/scatter.py#L24

References:
- https://developer.mozilla.org/en-US/docs/Tools/Accessibility_inspector/Simulation
- https://docs.generic-mapping-tools.org/6.1/gmtcolors.html#list-of-colors
- https://colorbrewer2.org/#type=qualitative&scheme=Set2&n=3

**Comment by ria-19** [comment]
Can I work on this issue?

**Comment by weiji14** [comment]
Hi @ria-19, this issue is already being worked on already at #1013. But you can find a list of other issues at https://github.com/GenericMappingTools/pygmt/contribute that we need help with. I recommend #908 which is not taken yet, though you're welcome to try another one too.

**Comment by ria-19** [comment]
Okay, Thank you!

**Comment by Nathandloria** [comment]
Hello! Recently my group @munzekm @cklima616 @noorbuchi and I took up work on #1070. We did this work for a class and another cycle has recently begun in that same class. We noticed that there was previously work being done on this issue, however progress seems to have slowed. Our team would love to take up work on this issue if possible. We really enjoyed this project and would love to continue contributing to pygmt!
